lib/iris/coords.py

```diff
@@ -1600,9 +1585,6 @@ def _sanitise_array(self, src, ndmin):
         # Ensure the array has enough dimensions.
         # NB. Returns the *same object* if result.ndim >= ndmin
         result = np.array(result, ndmin=ndmin, copy=False)
-        # We don't need to copy the data, but we do need to have our
-        # own view so we can control the shape, etc.
-        result = result.view()
```

Comment:
Isn't the reason for this to stop users messing with the points array 'by the back door'?
```
>>> d = DimCoord([1,2,3,4])
>>> d.points.shape = (2,2)
>>> print d
DimCoord(array([[1, 2],
       [3, 4]]), standard_name=None, units=Unit('1'))
```